### ISSUE TITLE
accumulated points reduces suddenly on versions 0.4.5 and 0.5.0.

### ISSUE THREAD

**Issue by harishbalachandran** [learningequality/kolibri#2252]
## Summary
The points accumulated by learners reduces suddenly on versions 0.4.5 and 0.5.0.

**This happens all of a sudden a learner with 6500 suddenly reduces to 4500 and 22,500 suddenly reduces to 12,500 etc.**

## System information
 - Version: 0.4.5 and 0.5.0
 - Operating system: Ubuntu 14.04LTS
 - Browser: Chrome

## How to reproduce
This is erratic and the best shot is to keep solving the exercise and observe the change in points.

## Screenshots
Sorry could not capture this since it happens at random intervals.

## Real-life consequences (anything community should be aware of, for instance how it affects your deployment)
1. Heartbroken learner who was religiously competing in the classroom to come first.
2. Disappointed coach.

**Comment by rtibbles** [comment]
@harishbalachandran what happens when the student refreshes the page after this? Does the points total then change to the correct total?

My only guess for why this might be happening is that the points total is cached on the initial load from the server, and somehow it gets reset to that total. But a page refresh would then restore the points displayed to the proper total.

**Comment by harishbalachandran** [comment]
The reduced points don't add up after refresh or even after logging in back to the next tab session.

**Comment by harishbalachandran** [comment]
@rtibbles, one more user story from a classroom: 
Students scored 500 points after solving 5 questions successively. These points were added in their overall total score at that time.

However when they automatically moved to next exercise then points didn't increase even after solving 5+ questions sequentially.

To resolve this we tried refreshing screen or again doing re-login. This resulted into students losing their previously scored points too.

**Comment by jonboiser** [comment]
@harishbalachandran We saw something similar in #1656. Could you check out the animation there and see if it matches your students' experience?

**Comment by navakk9** [comment]
@rtibbles - Harish explored this further, and saw that when kids master an exercise and move on to the next, 1000 points automatically gets added initially. 500 for current, and an additional 500 for the new exercise, which they navigated to by clicking on "go to next item". When refreshed, the 500 points are reduced which were extra initially.

**Comment by rtibbles** [comment]
Ah, that makes some sense. Have you been able to replicate locally also?

**Comment by harishbalachandran** [comment]
@rtibbles and @navakk9,  My sixth sense says and a wild guess is that :) 
Since the next exercise is also getting rewarded and it also carries the mastery state (Correct and incorrect tick marks) from the previous exercise, some how this gets captured as completed exercise on the coach report, but when the teacher tries to see the progress he/she stumbles upon this issue #2089.
I had noticed this when I navigated to the coach report to check the progress of the exercise that the learner had attempted. I will confirm the same once again tomorrow in detail.

I had forwarded a photo and 2 videos to Navya, it shows the exact problem, feel free to share it. The attachment is bulky to add here.
